### ISSUE TITLE
Action notices nonexistent tag

### ISSUE THREAD

**Issue by Plaenkler** [StephaneBour/actions-calver#14]
Release 2023.05 already exists. 2023.52 does not, of course. I can't explain how the problem comes about.

Affected repo: https://github.com/Plaenkler/GoST

```
 * [new tag]         2023.05    -> 2023.05
 * [new tag]         2023.52    -> 2023.52
Last release : 2023.52
Last hash : 3b372097b92d6099718bf45deaf807f1380c0b7e
Last major release : 2023.52
Next release : 2023.05
##  (2023-02-05)

* Merge pull request #93 from Plaenkler/92-secure-the-http-server-with-ssltls ([9f0d60a](https://github.com/Plaenkler/GoST/commit/9f0d60a)), closes [#93](https://github.com/Plaenkler/GoST/issues/93)
Create release : true
{
  "tag_name": "2023.05",
  "target_commitish": "main",
  "name": "2023.05",
  "body": "##  (2023-02-05)\n\n* Merge pull request #93 from Plaenkler/92-secure-the-http-server-with-ssltls ([9f0d60a](https://github.com/Plaenkler/GoST/commit/9f0d60a)), closes [#93](https://github.com/Plaenkler/GoST/issues/93)",
  "draft": false,
  "prerelease": false
}
{
  "message": "Validation Failed",
  "errors": [
    {
      "resource": "Release",
      "code": "already_exists",
      "field": "tag_name"
    }
  ],
  "documentation_url": "https://docs.github.com/rest/releases/releases#create-a-release"
}
```

**Comment by wosiu** [comment]
I've got the same issue. The minor part is not incremented during next release.

**Comment by wosiu** [comment]
I my case this seems to be the issue:
https://github.com/StephaneBour/actions-calver/blob/11869d66a40ed392908fa345dcb9774ae1b1848b/entrypoint.sh#L31

Secifically `grep "^20[^\-]*$"`, because it assumes my format starts with `20`, which is not true.

**Comment by wosiu** [comment]
I think we should introduce one more parameter to the `entrypoint.sh` script, which would accept regexp for this `grep` with default `"^20[^\-]*$" ` for backward compatibility. @StephaneBour let me know if it sounds good for you and I can create a PR.

**Comment by wosiu** [comment]
Actually, here it is: https://github.com/StephaneBour/actions-calver/pull/16

**Comment by StephaneBour** [comment]
Thank you a lot @wosiu

**Comment by miigotu** [comment]
Fixed in #16
